### ISSUE TITLE
Improve accessible naming for resource icons in project pages

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#6655]
### Dependency
- #6653

### Overview
We need to correct the accessible name currently assigned to each resource link in the **Resources** section of each project page so that it matches WCAG guidelines

### Details 
An accessible name is a short string of text that identifies a user interface element and gives users of assistive technology a label for it.  The "alt-text" attribute is used to provide an accessible name for an img element, however the "alt-text" attribute is not applicable to an svg element.  The wiki page https://github.com/hackforla/website/wiki/How-to-make-SVGs-and-other-images-WCAG-compliant describes methods for providing an accessible name to svg elements, however there are instances in which additional measures are required.  

If an element (in this case, the `<a>` element wrapping each resource icon) does not have an accessible name supplied, the user agent will calculate an accessible name by concatenating the accessible names of child nodes.  This can lead to accessible names which contain duplicate text; e.g., the accessible name of the GitHub link is "GitHub GitHub".  This problem and a solution are depicted in the Resources/Instructions section below. 

### Action Items
- [x] Open `_includes\resource-card.html` in your IDE
- [x] Bring up a project page, for example `https://www.hackforla.org/projects/311-data` then use either the accessibility tree or a browser screen reader extension to confirm that the accessible name for the resource links is duplicated.  For example the accessible name of the GitHub link is "GitHub GitHub".  
- [x] Update the HTML to correct the accessible name for the resource links.  
   - [x] One solution is to provide the links with an accessible name using the attribute `aria-labelledby` that references the id of another element with the same name, such as the <h4> element.  
- [x] Using docker and using either the accessibility tree or the screen reader extension, confirm that the accessible name of the resource links as well as the accessible name of the resource icons is correct.  E.g., the accessible name of both the GitHub resource link and image should be "GitHub"
- [x] Confirm that the behavior and appearance of the page is unchanged

### Resources/Instructions

<details><summary>HTML and accessibility tree depicting a duplicate accessible name "GitHub GitHub" on the link wrapping an svg element</summary>

![image](https://github.com/hackforla/website/assets/5314153/ee9bd117-a4d8-47e1-9199-0a96e59e755a)

</details> 

Sample Project page https://www.hackforla.org/projects/311-data
https://github.com/hackforla/website/wiki/How-to-make-SVGs-and-other-images-WCAG-compliant
https://www.w3.org/TR/using-aria/
https://canvas.workday.com/accessibility/the-accessibility-tree/#how-to-view-the-accessibility-tree

- This issue was created from #4242

**Comment by terrencejihoonjung** [comment]
ETA: 06/06/24, 11:59pm PST
Availability: 9am - 5pm PST, Mon-Fri
